std/traits.d

```diff
@@ -8087,3 +8089,152 @@ enum isCopyable(S) = is(typeof(
     static assert(isCopyable!int);
     static assert(isCopyable!(int[]));
 }
+
+// Gets the type of a value, or just the type passed if it's a type.
+private template TypeOf(T...)
+{
+    static if (is(T[0] U))
+        alias TypeOf = U;
+    else static if (is(typeof(T[0]) U))
+        alias TypeOf = U;
+}
+
+@safe unittest
+{
+    static assert(is(TypeOf!3 == int));
+    static assert(is(TypeOf!int == int));
+    static assert(is(TypeOf!([]) == void[]));
+    static assert(!is(TypeOf!(mixin(__MODULE__))));
+    static assert(is(TypeOf!TypeOf));
+}
+
+/**
+ * Get a list of the overloads for the given callable, in the order in which they are defined in the code.
+ *
+ * Params:
+ *     func = The callable whose overload set to return.
+ * Returns:
+ *     A tuple of the overloads for the given callable.
+ */
+template getOverloads(func...)
+if (func.length == 1 && isCallable!func)
+{
+    enum isModuleOrPackage(alias s) = !is(typeof(s));
+    alias Parent = __traits(parent, func);
+
+    static if (is(Parent) || isModuleOrPackage!Parent)
+        alias getOverloads = AliasSeq!(__traits(getOverloads, Parent, __traits(identifier, func)));
```

Comment:
`AliasSeq` is not required anymore to alias `__traits` results.

Reply:
Also don't forget that `getOverloads` has an optional 3rd parameter since a few releases. It's a bool and it allows to include templates (esp. function templates).